src/platform/forms-system/src/js/web-component-patterns/datePatterns.jsx

```diff
@@ -43,20 +49,15 @@ const currentOrPastDateUI = options => {
     'ui:options': {
       ...uiOptions,
     },
+    'ui:confirmationField': ({ formData }) => ({
```

Comment:
new implementation looks good!

I have one more suggestion - rather than committing all these patterns to use ui:confirmationField before the feature is released, an idea is to do something similar that we do in the ReviewFieldTemplate (lines 55-64) we automatically assign `ui:widget` when needed for dates. e.g.
(while going through the fields)
```
uiSchema &&
!uiSchema['ui:confirmationField'] && (
uiSchema['ui:widget'] === 'date' ||
uiSchema['ui:webComponentField']?.name === 'VaMemorableDate' || uiSchema['ui:webComponentField']?.name === 'VaDate')
```
and if this is true, then we can set
```
uiSchema['ui:confirmationField'] = // expected code
```

I think something like this would help so we don't have to define ui:confirmationField on a bunch of patterns before it is generally available.

Reply:
Excellent suggestion, I'll implement this.

Reply:
Done!